bqskit/ir/circuit.py

```diff
@@ -2209,9 +2215,12 @@ def get_unitary_and_grad(
     def instantiate(
         self,
         target: StateLike | UnitaryLike,
-        method: str | None = None,
+        method: str | Instantiater | None = None,
         multistarts: int = 1,
         seed: int | None = None,
+        start_gen: MultiStartGenerator = RandomStartGenerator(),
```

Comment:
```suggestion
        multistart_gen: MultiStartGenerator = RandomStartGenerator(),
```

(You'll need to change its usage below too)